openmetadata-service/src/main/java/org/openmetadata/service/search/elasticsearch/ElasticSearchClient.java

```diff
@@ -366,6 +366,13 @@ public Response previewSearch(
   public Response doSearch(
       SearchRequest request, SubjectContext subjectContext, SearchSettings searchSettings)
       throws IOException {
+    // Override with domain query
+    if (request.getDomains().stream().findFirst().isPresent()) {
+      String domain = request.getDomains().get(0).getFullyQualifiedName();
+      String newQuery = request.getQuery() + " AND (domain.fullyQualifiedName:\""+domain+"\")";
```

Comment:
we need to stop thinking that we will be using queryString, currently queryString being deprecated in our searches we already use multimatch, please check that and adjust accodingly don't want to pass this in this format